### PR TITLE
Fix broken operation priorities for traced stores

### PR DESCRIPTION
**Pull request by ilyaagarkov**
I have used pattern from patronum/debug for tracing store but not sure I did everything right 

As I see this changes fix problems described here https://github.com/effector/logger/issues/100

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -244,31 +245,41 @@ function copy<T>(a: T): T {
...
-  graphite($store).seq.push(
-    step.compute({
-      fn(data, scope, stack) {
-        traceStoreChange({
-          type: 'store',
-          name,
-          before: scope.trace.before,
-          current: data,
-        });
+        before = copy(scope.state.current)
```
Comment (ilyaagarkov) [inline src/index.ts:256]: I'm not sure how we can use previous value with `createNode`

File: src/index.ts [hunk 2]
```diff
@@ -36,6 +36,7 @@ import {
   TraceEffectRun,
 } from './types.h';
 import { Root } from './view';
+import { createNode, Stack } from "effector/effector.cjs";
```
Comment (zerobias) [inline src/index.ts:39]: Dont use direct imports from specific bundles, please

**Review by zerobias — CHANGES_REQUESTED** [review]
The idea itself should work, please fix imports

File: src/index.ts (hunk 2 above)
Comment (ilyaagarkov) [reply src/index.ts:39]: IDEal auto imports sometimes lets down

**Review by sergeysova — APPROVED** [review]
Looks good.

**Review by zerobias — APPROVED** [review]

**Pull request merged**